tests/integration/ovs_test.py

```diff
@@ -204,12 +193,18 @@ def test_ovs_remove_port(bridge_with_ports):
 
 @contextmanager
 def disable_nm_ovs_plugin():
```

Comment:
Since the two plugin helpers are identical except for the plugin RPM, it would be nice to have one context manager that gets the RPM name as argument to reduce the duplication IMHO. Then you also do not need the constants for the commands but can use them directly in the function.

Reply:
It looks good. Where should I put this new context manager? Thanks!

Reply:
How about `nmstate/tests/integration/testlib/plugin.py`? Needs to be created. IMHO it could also be directly in `__init__.py` but it might be that @EdDev has arguments against it.

Reply:
Done, thanks!